docs/articles/documentation/using-testcafe/common-concepts/live-mode.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: docs
+title: Live Mode
+permalink: /documentation/using-testcafe/common-concepts/live-mode.html
+---
+# Live Mode
+
+Live mode keeps the TestCafe process and browsers opened the whole time you are working on tests. Changes you make in code immediately restart the tests. That is, live mode allows you to see test results instantly.
+
+![Live mode demonstrated in action](../../../images/testcafe-live.gif)
+
+> This feature replaces the `testcafe-live` module. This module is now deprecated.
+
+## How to Enable Live Mode
+
+Use the [-L (--live)](../command-line-interface.md#-l---live) flag to enable live mode from the command line interface.
+
+```sh
+testcafe chrome tests/test.js -L
+```
+
+In the API, create a [live mode runner](../programming-interface/livemoderunner.md) with the [testcafe.createLiveModeRunner](../programming-interface/testcafe.md#createlivemoderunner) function and use it instead of a [regular test runner](../programming-interface/runner.md).
+
+```js
+const createTestCafe = require('testcafe');
+let testcafe         = null;
+
+createTestCafe('localhost', 1337, 1338)
+    .then(tc => {
+        testcafe         = tc;
+        const liveRunner = testcafe.createLiveModeRunner();
+
+        return liveRunner
+            .src('tests/test.js')
+            .browsers('chrome')
+            .run();
+    })
+    .then(() => {
+        testcafe.close();
+    });
+```
+
+## How Live Mode Works
+
+When you run tests with live mode enabled, TestCafe opens the browsers, runs tests there, shows the reports and waits for your further actions.
```

Comment:
When you run tests with live mode enabled, TestCafe opens the browsers, runs the tests, shows the reports and waits for further actions.